front-end/src/components/CommentList.tsx

```diff
@@ -1,54 +1,20 @@
-import React, { useEffect, useState } from 'react';
-import { Col, Button, ListGroup, ListGroupItem, ListGroupItemHeading, ListGroupItemText, Card, CardTitle, Container, Row } from 'reactstrap';
-import { UserLogin } from '../types/UserLogin';
-import CreateEditPostModal from '../components/CreateEditPostModal';
-import PostListItem from '../components/PostListItem';
-import { Post } from '../types/Post';
-import { Author } from '../types/Author';
+import { Col, Container } from 'reactstrap';
 import { PostComment } from '../types/PostComment';
 import CommentListItem from './CommentListItem';
-import { AxiosWrapper } from '../helpers/AxiosWrapper';
 
 interface Props {
-  postId: string,
-  postAuthor: Author,
-  loggedInUser: UserLogin | undefined,
```

Comment:
Can you leave this in. I'm going to need these to pass to CommentListElement for liking comments.

Reply:
Done.